jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/newcase/NewCaseInstanceViewImpl.java

```diff
@@ -99,21 +118,35 @@ public void cleanForm() {
         caseTemplatesList.setValue("");
         caseTemplatesList.getElement().focus();
         caseTemplatesList.enable();
+        ownerNameInput.setValue("");
         clearErrorMessages();
     }
 
     private boolean validateForm() {
         clearErrorMessages();
 
+        boolean valid = true;
+
         if (isNullOrEmpty(caseTemplatesList.getValue())) {
             caseTemplatesList.getElement().focus();
             definitionNameHelp.setTextContent(translationService.format(PLEASE_SELECT_CASE_DEFINITION));
             caseDefinitionNameGroup.setValidationState(ValidationState.ERROR);
-            return false;
-        } else {
+            valid = false;
+        }
+
+        if (isNullOrEmpty(ownerNameInput.getValue())) {
+            ownerNameInput.focus();
+            ownerNameHelp.setTextContent(translationService.format(PLEASE_PROVIDE_CASE_OWNER));
+            ownerNameGroup.setValidationState(ValidationState.ERROR);
+            valid = false;
+        }
```

Comment:
Agreed, by default the current user should be assigned as the case owner. That is in place here, the current user is defined as the value for the owner field. That way not only it still going to assign the new case instance to the current user but also allow that if there is need to assign to a different user, that could be done by simply providing a different user at the start case popup.